tests/automatic_differentiation/ad_argon.i

```diff
@@ -1,7 +1,7 @@
 dom0Scale=1e-3
 
 [GlobalParams]
-  offset = 20
+  offset = 30
   # offset = 0
```

Comment:
Several instances of lingering commented lines from troubleshooting in this file....could this be cleaned up since the test has been re-golded? Or if it needs to remain to illustrate something, include a descriptive comment.

Reply:
I just commented on the first instance, but this happens several times in this file.

Reply:
Yeah, sorry - I leave a mess sometimes. I'll clean this up.